src/hub.html

```diff
@@ -453,48 +453,48 @@
                 rotation
                 pitch-yaw-rotator
                 set-yxz-order
-                sound__teleport_start="positional: false; src: #sound_asset-teleport_start; on: nothing; poolSize: 2; loop: true;"
+                sound__teleport_start="positional: false; src: #sound_asset-teleport_start; on: nothing; poolSize: 2; loop: true; positional: false;"
```

Comment:
This and the rest of the sounds on the player rig already have `position: false`.